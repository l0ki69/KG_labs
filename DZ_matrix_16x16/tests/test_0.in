1111110000000000
1111100000000000
1111000000000000
1110000011110000
1100000000000000
1000000000000000
0000000000001110
0000001001000000
0000000011100011
0001000001000001
0001000000000000
0000111000000000
0000000000011000
0001000000000000
0000000111000000
0000000000000000