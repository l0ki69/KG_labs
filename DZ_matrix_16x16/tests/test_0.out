Отметка #0. Площадь 21 - земля
Отметка #1. Площадь 4 - цель
Отметка #2. Площадь 3 - цель
Отметка #3. Площадь 1 - шум или аномалия
Отметка #4. Площадь 5 - земля
Отметка #5. Площадь 3 - цель
Отметка #6. Площадь 2 - шум или аномалия
Отметка #7. Площадь 3 - цель
Отметка #8. Площадь 2 - шум или аномалия
Отметка #9. Площадь 1 - шум или аномалия
Отметка #10. Площадь 3 - цель
